docs/details/dashboards/dashboard-inventory.md

```diff
@@ -118,6 +118,9 @@ Click the downward arrow to view cluster details, including the services running
 
 ![!image](../../_images/PMM_Inventory_cluster_view_details.png)
 
+Furthermore, you can filter the clusters by criteria such as cluster name, Status, Service name, Node name, Monitoring, Address, and Port. 
```

Comment:
Maybe just capitalize Cluster name (and or lowercase other fields whatever is preferred). Otherwise no other comments from me.